--- origsrc/pixman-0.10.0/configure.ac	2008-03-27 09:01:45.000000000 -0500
+++ src/pixman-0.10.0/configure.ac	2008-05-13 21:27:33.203125000 -0500
@@ -81,7 +81,7 @@
 m4_define([lt_revision], [pixman_micro])
 m4_define([lt_age], [pixman_minor])
 
-LT_VERSION_INFO="lt_current:lt_revision:lt_age"
+PIXMAN_VERSION_INFO="lt_current:lt_revision:lt_age"
 
 PIXMAN_VERSION_MAJOR=pixman_major()
 AC_SUBST(PIXMAN_VERSION_MAJOR)
@@ -90,7 +90,7 @@
 PIXMAN_VERSION_MICRO=pixman_micro()
 AC_SUBST(PIXMAN_VERSION_MICRO)
 
-AC_SUBST(LT_VERSION_INFO)
+AC_SUBST(PIXMAN_VERSION_INFO)
 
 # Check for dependencies
 #PKG_CHECK_MODULES(DEP, x11)
--- origsrc/pixman-0.10.0/pixman/Makefile.am	2008-03-27 06:05:16.000000000 -0500
+++ src/pixman-0.10.0/pixman/Makefile.am	2008-05-13 21:27:48.734375000 -0500
@@ -1,5 +1,5 @@
 lib_LTLIBRARIES = libpixman-1.la
-libpixman_1_la_LDFLAGS = -version-info $(LT_VERSION_INFO)
+libpixman_1_la_LDFLAGS = -version-info $(PIXMAN_VERSION_INFO) -no-undefined
 libpixman_1_la_LIBADD = @DEP_LIBS@ -lm
 libpixman_1_la_SOURCES =		\
 	pixman.h			\
